src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -140,6 +140,10 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
     $(button).attr('title', tooltipString).tooltip('fixTitle');
 }
 
+function isInExpandMode(button) {
+    return $(button).html().trim().startsWith('Expand');
+}
```

Comment:
This can be moved to within the `expandOrCollapsePanels` function.

Reply:
We were using this statement lots of time so you had asked to extract it? https://github.com/TEAMMATES/teammates/pull/8446#discussion_r175139320